Automattic/woocommerce-payments
# Move hooks out of MultiCurrency class constructors into own init_hooks methods

Fixes #7258

#### Changes proposed in this Pull Request

* Add new `init_hooks` methods to Multi-Currency related classes.
* Add calls to the new `init_hooks` methods when classes are created and/or initialized.
* Update tests to account for new `init_hooks` methods.

#### Testing instructions

* Tests should pass.
* You should be able to use Multi-Currency with no issues:
  * Access Multi-Currency settings in the admin.
  * Use the Multi-Currency widget on the front end.

-------------------
